src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/cache/IdamCachedClient.java

```diff
@@ -31,20 +37,32 @@ public IdamCachedClient(
         this.users = users;
         this.accessTokenCache = Caffeine.newBuilder()
             .expireAfter(accessTokenCacheExpiry)
+            .removalListener(this::onCachedIdamTokenRemoval)
+            .build();
+
+        this.userDetailsCache =  Caffeine.newBuilder()
+            .maximumSize(200)
```

Comment:
is there a reason this value is high. We would have 5 keys /values at any given time ?

Reply:
it will be configurable. but 5 is not enough as we better give some buffer to maintain the cache